indexer/src/metrics.rs

```diff
@@ -0,0 +1,55 @@
+use actix_web::{get, App, HttpServer, Responder};
+use lazy_static::lazy_static;
+use prometheus::{Encoder, IntCounter, IntGauge, Opts};
+use tracing::info;
+
+lazy_static! {
+    pub(crate) static ref LATEST_BLOCK_HEIGHT: IntGauge = try_create_int_gauge(
+        "indexer_explorer_latest_block_height",
+        "Height of last processed block"
+    )
+    .unwrap();
+    pub(crate) static ref BLOCK_COUNT: IntCounter =
+        try_create_int_counter("indexer_explorer_block_count", "Number of indexed blocks").unwrap();
+}
```

Comment:
dropped the `-lake` prefix from the metrics since we've done the same everywhere else